scrapscript.py

```diff
@@ -1037,7 +1107,7 @@ def eval_list(env: Env, exp: Object) -> typing.List[Object]:
 
 
 def make_bool(x: bool) -> Object:
-    return Symbol("true" if x else "false")
+    return Tag(Tags({"true": Hole(), "false": Hole()}), "true" if x else "false", Hole())
```

Comment:
this should probably be a global constant at this point and we can use in `eval_bool` as well